test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/Creating/CreateResourceWithRelationshipsTests.cs

```diff
@@ -0,0 +1,219 @@
+                    {
+                        color = new
+                        {
+                            data = new
+                            {
+                                type = "rgbColors",
+                                id = group.Color.StringId
+                            }
+                        }
+                    }
+                }
+            };
+
+            var route = "/workItemGroups";
+
+            // Act
+            var (httpResponse, responseDocument) = await _testContext.ExecutePostAsync<ErrorDocument>(route, requestBody);
+            
+            // Assert
+            httpResponse.Should().HaveStatusCode(HttpStatusCode.BadRequest);
+            
+            responseDocument.Errors.Should().HaveCount(1);
+            responseDocument.Errors[0].StatusCode.Should().Be(HttpStatusCode.BadRequest);
+            responseDocument.Errors[0].Title.Should().Be("Relationships are not supported when using MongoDB.");
+            responseDocument.Errors[0].Detail.Should().BeNull();
+        }
+
+        [Fact]
+        public async Task Cannot_create_OneToOne_relationship_from_dependent_side()
+        {
+            // Arrange
+            var color = _fakers.RgbColor.Generate();
+            var group = _fakers.WorkItemGroup.Generate();
+            group.Id = ObjectId.GenerateNewId().ToString();
+            group.Color = color;
+            color.Group = group;
+
+            await _testContext.RunOnDatabaseAsync(async db =>
+            {
+                await db.GetCollection<RgbColor>().InsertOneAsync(color);
+            });
+
+            var requestBody = new
+            {
+                data = new
+                {
+                    type = "rgbColors",
+                    id = ObjectId.GenerateNewId().ToString(),
+                    relationships = new
+                    {
+                        group = new
+                        {
+                            data = new
+                            {
+                                type = "workItemGroups",
+                                id = color.Group.StringId
+                            }
+                        }
+                    }
+                }
+            };
+
+            var route = "/rgbColors";
+
+            // Act
+            var (httpResponse, responseDocument) = await _testContext.ExecutePostAsync<ErrorDocument>(route, requestBody);
+            
+            // Assert
+            httpResponse.Should().HaveStatusCode(HttpStatusCode.BadRequest);
+            
+            responseDocument.Errors.Should().HaveCount(1);
+            responseDocument.Errors[0].StatusCode.Should().Be(HttpStatusCode.BadRequest);
+            responseDocument.Errors[0].Title.Should().Be("Relationships are not supported when using MongoDB.");
+            responseDocument.Errors[0].Detail.Should().BeNull();
+        }
+
+        [Fact]
+        public async Task Cannot_create_relationship_with_include()
```

Comment:
This one can be removed too. Creating a resource with a relationship is not possible, so doing that and then fetching the related data is not possible either.